process/testdata/executingSCCall.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
can you add also this transaction it your test data, it's a complex one that had failed https://explorer.multiversx.com/transactions/a93ab7b43150786e2a9639137d3ba13c31273b2f82b9555b1fb2a76b3d711151

Reply:
sure, will add

Reply:
done